### DISCUSSION TITLE
[7/27/2023] New Mojo Release!

### DISCUSSION THREAD

**Discussion by goldiegadde** [modular/modular#481] [Announcements]
Hey everyone 👋 ,

A new Mojo🔥 release has been rolled out to Playground a few hours back! To see what's new, [check out the changelog](https://docs.modular.com/mojo/changelog.html).

If you have an active session in the Mojo Playground, you'll need to restart it to get the updated build.
Thanks,
Goldie

**Comment by drunkwcodes** [comment]
Loop unrolling has never been that easy!
